pkg/kubelet/metrics/metrics.go

```diff
@@ -719,6 +723,24 @@ var (
 		},
 	)
 
+	MemoryManagerPinningRequestTotal = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           MemoryManagerPinningRequestsTotalKey,
+			Help:           "The number of memory pages allocations which required pinning.",
+			StabilityLevel: metrics.ALPHA,
+		})
+
+	// MemoryManagerPinningErrorsTotal tracks the number of times the pod spec required the memory manager to pin memory pages, but the allocation failed
+	MemoryManagerPinningErrorsTotal = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           MemoryManagerPinningErrorsTotalKey,
+			Help:           "The number of memory pages allocations allocations which required pinning failed.",
```

Comment:
```suggestion
			Help:           "The number of memory pages allocations which required pinning failed.",
```

TBH this reads a bit hard to me.

Reply:
Done. Thank you